src/main/java/com/microsoft/graph/core/IBaseClient.java

```diff
@@ -86,4 +86,9 @@
      * Validates this client
      */
     void validate();
+    
+    /*
+     * Shuts down the executors.
+     */
+    void shutDown();
```

Comment:
minor: spelling shutdown

Reply:
yes I hoped my correction would be applied everywhere without having to say it!